src/components/ClusterManager.js

```diff
@@ -261,7 +261,7 @@ export class NewClusterModal extends PureComponent {
                 numberOfWorkers: v ? 2 : 0,
                 numberOfPreemptibleWorkers: 0
               })
-            }, ' Configure as Spark cluster')
+            }, ' Configure as Spark notebook runtime')
```

Comment:
This one should remain 'cluster', since this is the config option for creating an actual cluster.